lib/logger.js

```diff
@@ -121,6 +121,29 @@ var streamConverter = {
 };
 var streamConverterList = Object.keys(streamConverter);
 
+Logger.prototype._addErrorHandler = function() {
+    var self = this;
+    self._logger.on('error', function(err, failedStream) {
+        // If some fatal error occurred in one of the logger streams,
+        // we can't do much, so ignore.
+
+        if (err.code === 'ENOSPC') {
```

Comment:
@gwicke I was thinking about that too. On one hand, we don't wanna stop logging if the error is recoverable, like some intermittent write failure. On the other hand, we wanna get rid of the logger for irrecoverable error.. I'm not sure where in the spectrum we wanna be...